sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentServiceRequest.java

```diff
@@ -26,8 +26,6 @@
  * This is core Transport/Connection agnostic request to the Azure Cosmos DB database service.
  */
 public class RxDocumentServiceRequest implements Cloneable {
-    private static final char PREFER_HEADER_SEPERATOR = ';';
-    private static final String PREFER_HEADER_VALUE_FORMAT = "%s=%s";
```

Comment:
What are scenarios for it?

Reply:
Unknown scenarios, but this already exists in the SDK, and there is a unit test as well. Didn't want to break that. 
https://github.com/Azure/azure-sdk-for-java/pull/10741/files#diff-7011a46947ace6e46648f8c4027e602eL414
So to keep the functionality intact, I am re-using the logic we already have.

Reply:
I am thinking if this is not exposed through public api, we should drop the generic support and only have support for minimalResponseContent scenario

Reply:
I like that idea @moderakh . I will get rid of it.